build.sc

```diff
@@ -5,7 +5,7 @@ import de.tobiasroeser.mill.vcs.version._
 import mill._, scalalib._, publish._
 import mill.scalalib.api.Util.scalaNativeBinaryVersion
 
-val millVersions       = Seq("0.9.12", "0.10.12", "0.11.0")
+val millVersions       = Seq("0.9.12", "0.10.12", "0.11.5")
```

Comment:
This change effectively means, you are no longer supporting Mill 0.11.0 - 0.11.4, as Mill does not guarantee any forward binary compatibilities. 

```suggestion
val millVersions       = Seq("0.9.3", "0.10.0", "0.11.0")
```